app/imports/api/methods/commentsMethods.js

```diff
@@ -0,0 +1,53 @@
+import { Meteor } from 'meteor/meteor'
+
+import { Applications } from '/imports/collections/applications.js';
+import { Comments } from '/imports/collections/comments.js';
+
+import { AccountsCommon } from 'meteor/accounts-base';
+
+export const submitComment = new ValidatedMethod({
+  name: 'comment.submit',
+
+  validate: new SimpleSchema({
+    applicationId: {
+      type: String,
+    },
+    comment: {
+      type: String,
+    },
+  }).validator(),
+
+  run({ applicationId, comment }) {
+    if (!Roles.userIsInRole(this.userId, ['committee', 'admin'])) {
+      throw new Meteor.Error('Coments.methods.submit.not-allowed', 'Must be committee to do this.');
+    }
+
+    let user = Meteor.users.findOne(this.userId);
+
+    Comments.insert({
+      userId: this.userId,
+      email: user.emails[0].address,
+      applicationId: applicationId,
+      content: comment,
+    });
+  }
+});
+
+export const deleteComment = new ValidatedMethod({
+  name: 'comment.delete',
+
+  validate(commentId) {
+    check(commentId, String);
+  },
+
+  run(commentId) {
+    if (!Roles.userIsInRole(this.userId, ['committee', 'admin'])) {
```

Comment:
You are misunderstanding me. at the moment any committee user with the comment ID can call this method and delete a comment even if they don't own it. That is wrong. (And admin should be allowed to delete all commments)